AppiumForMac/Server/Handlers/AfMHandlers.m

```diff
@@ -543,7 +543,7 @@ - (AppiumMacHTTPJSONResponse *)post_elements:(NSString*)path data:(NSData*)postD
                     [elements addObject:[NSDictionary dictionaryWithObject:myKey forKey:@"ELEMENT"]];
                 }
                 
-                return [AppiumMacHTTPJSONResponse responseWithJson:nil status:kAfMStatusCodeNoSuchElement session:session.sessionId];
+                return [AppiumMacHTTPJSONResponse responseWithJson:elements status:kAfMStatusCodeSuccess session:session.sessionId];
             }
         }
```

Comment:
I assume it should also return success and an empty array after `matches.count > 0` block